README.md

```diff
@@ -1,4 +1,4 @@
-# Woocommerce HT PayWay
```

Comment:
missing_readme_header.

I think we need to add docblock by following docs: https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#markdown